lib_eio/buf_read.mli

```diff
@@ -107,6 +107,48 @@ val string : string -> unit parser
 
     @raise Failure if [s] is not a prefix of the stream. *)
 
+(** Big endian parsers *)
+module BE : sig
+  val uint16 : int parser
+  (** [uint16] returns the next 2 bytes as the lower 16 bits of an [int] in big-endian byte order *)
+
+  val uint32 : int32 parser
+  (** [uint32] returns the next 4 bytes as an [int32] in big-endian byte order *)
+
+  val uint48 : int64 parser
+  (** [uint48] returns the next 6 bytes as an [int48] in big-endian byte order *)
```

Comment:
```suggestion
  (** [uint48] returns the next 6 bytes as an [int64] in big-endian byte order *)
```
Assuming this is supposed to be an OCaml type. Though it seems unnecessary to mention it, as it's already in the function signature.
```suggestion
  (** [uint48] parses 6 bytes as a 48-bit unsigned big-endian integer. *)
```